vng_api_common/serializers.py

```diff
@@ -188,12 +188,22 @@ def to_representation(self, instance) -> dict:
         ret = OrderedDict()
         fields = self._readable_fields
 
+        all_empty = True
         for field in fields:
             attribute = instance[field.field_name]
             if attribute is None:
                 ret[field.field_name] = None
             else:
                 ret[field.field_name] = field.to_representation(attribute)
+
+            if bool(attribute):
+                all_empty = False
+
+        # If the Gegevensgroep is nullable and all attributes are empty,
```

Comment:
Can we stick to keeping a consistent output, but allowing `null` for input? In all other places, "empty" means you get the gegevensgroep back, but with empty fields within. I'd like to keep this predictable.

Reply:
So in the case of a `null` as input, the response should still contain the entire gegevensgroep back (with `null`/empty strings as values)?